packages/components/src/custom-select-control/README.md

```diff
@@ -131,6 +131,20 @@ Start opting into the unconstrained width style that will become the default in
 -   Required: No
 -   Default: `false`
 
+#### onMouseOver
```

Comment:
I guess it makes sense to add these 2 functions to the documentation for now. In future refactors of the component, I guess we won't need to to do if we use the `WordPressComponentProps` type, since it inherits all standard HTML attributes/props.